java/arcs/core/storage/driver/Volatile.kt

```diff
@@ -262,7 +262,7 @@ class VolatileDriverProviderFactory : DriverProvider {
     /** Clears everything from storage. */
     fun clear() = synchronized(lock) { entries.clear() }
 
-    /** Clears everything from storage. */
+    /** Returns the number of [VolatileEntry] in storage. */
     fun size() = synchronized(lock) { entries.size }
```

Comment:
I think this is also unused and can be removed.

Reply:
Done.